assets/src/containers/ResourcesAccessed.js

```diff
@@ -67,6 +67,7 @@ function ResourcesAccessed (props) {
   // this is the filter setting last saved by the user
   const [userSavedFilterSetting, setUserSavedFilterSetting] = useState(resourceGradeFilter)
   const [resourceAccessData, setResourceAccessData] = useState('')
+  const [resourcesLimit, setResourcesLimit] = useState(0)
```

Comment:
There may be a bug here, I'm looking into it. I think you meant for the initial state to be `undefined`.

Reply:
I do think this should be `undefined`, but I don't think it's the direct cause of the issue. See conversation elsewhere.

Reply:
That makes sense. Probably also using `null` could work out here for what I'm intending, just would have to check that.